extensions/ql-vscode/src/vscode-tests/minimal-workspace/databases.test.ts

```diff
@@ -391,145 +378,146 @@ describe("databases", () => {
       );
-      expect(await db.isAffectedByTest("/path/to/dir")).to.false;
+      expect(await db.isAffectedByTest("/path/to/dir")).toBe(false);
     });
 
     it("should return false for testproj database for prefix directory", async () => {
-      sandbox.stub(fs, "stat").resolves(directoryStats);
+      jest.spyOn(fs, "stat").mockResolvedValue(directoryStats);
       const db = createMockDB(
         sourceLocationUri(),
         Uri.file("/path/to/dir/dir.testproj"),
       );
       // /path/to/d is a prefix of /path/to/dir/dir.testproj, but
       // /path/to/dir/dir.testproj is not under /path/to/d
-      expect(await db.isAffectedByTest("/path/to/d")).to.false;
+      expect(await db.isAffectedByTest("/path/to/d")).toBe(false);
     });
 
     it("should return true for testproj database for test file", async () => {
-      sandbox.stub(fs, "stat").resolves(fileStats);
+      jest.spyOn(fs, "stat").mockResolvedValue(fileStats);
       const db = createMockDB(
         sourceLocationUri(),
         Uri.file("/path/to/dir/dir.testproj"),
       );
-      expect(await db.isAffectedByTest("/path/to/dir/test.ql")).to.true;
+      expect(await db.isAffectedByTest("/path/to/dir/test.ql")).toBe(true);
     });
 
     it("should return false for non-existent test file", async () => {
-      sandbox.stub(fs, "stat").throws("Simulated Error: ENOENT");
+      jest.spyOn(fs, "stat").mockImplementation(() => {
+        throw new Error("Simulated Error: ENOENT");
+      });
       const db = createMockDB(
         sourceLocationUri(),
         Uri.file("/path/to/dir/dir.testproj"),
       );
-      expect(await db.isAffectedByTest("/path/to/dir/test.ql")).to.false;
+      expect(await db.isAffectedByTest("/path/to/dir/test.ql")).toBe(false);
     });
 
     it("should return false for non-testproj database for test file", async () => {
-      sandbox.stub(fs, "stat").resolves(fileStats);
+      jest.spyOn(fs, "stat").mockResolvedValue(fileStats);
       const db = createMockDB(
         sourceLocationUri(),
         Uri.file("/path/to/dir/dir.proj"),
       );
-      expect(await db.isAffectedByTest("/path/to/dir/test.ql")).to.false;
+      expect(await db.isAffectedByTest("/path/to/dir/test.ql")).toBe(false);
     });
 
     it("should return false for testproj database not matching test file", async () => {
-      sandbox.stub(fs, "stat").resolves(fileStats);
+      jest.spyOn(fs, "stat").mockResolvedValue(fileStats);
       const db = createMockDB(
         sourceLocationUri(),
         Uri.file("/path/to/dir/dir.testproj"),
       );
-      expect(await db.isAffectedByTest("/path/to/test.ql")).to.false;
+      expect(await db.isAffectedByTest("/path/to/test.ql")).toBe(false);
     });
   });
 
-  describe("findSourceArchive", function () {
+  describe("findSourceArchive", () => {
     // not sure why, but some of these tests take more than two seconds to run.
-    this.timeout(5000);
+    jest.setTimeout(5000);
```

Comment:
I don't think this does anything, or at least not the thing we expect; `jest.setTimeout` is on a per-file basis (and the default is also 5 seconds).